package.json

```diff
@@ -66,7 +66,7 @@
     "@bbc/psammead-styles": "^4.3.0",
     "@bbc/psammead-test-helpers": "^3.1.2",
     "@bbc/psammead-timestamp": "^2.2.24",
-    "@bbc/psammead-timestamp-container": "^2.6.21",
+    "@bbc/psammead-timestamp-container": "^2.7.3",
```

Comment:
Psammead's main `package-lock.json` should be updated too since the version of `timestamp-container` changed. It will automatically update after you run `npm run install:packages`

Reply:
Version `2.7.3` will only be deployed after this PR is merged, we should not update `psammead/package.json` here.

Reply:
But will Jenkins allow merging this since now it is complaining that `package.json` and `package-lock.json` are not in sync?

Reply:
the formatDuration utility function has been implemented and merged in a different PR #3114 , that functionality is now being pulled in to this PR.